src/blogs/performance.md

```diff
@@ -0,0 +1,71 @@
+---
+
+title: Monitoring DreamHack - the World's Largest Digital Festival
+author: rahul
+author_info: Software Engineer at MayaData, working on Cloud Native Tech
+tags: devops
+image: img1.png
```

Comment:
Can we get this image directly with help of title text like we discussed in the standup?

Reply:
Added slug and updated the images.